docs/src/main/asciidoc/grpc-service-consumption.adoc

```diff
@@ -243,6 +241,19 @@ quarkus.grpc.clients.hello.ssl.key=tls/client.key
 quarkus.grpc.clients.hello.ssl.trust-store=tls/ca.pem
 ----
 
+=== Client Deadlines
+
+It's always reasonable to set a deadline for a gRPC client, i.e. to specify a duration of time after which the RPC times out and the client receives the status error `DEADLINE_EXCEEDED`.
```

Comment:
While deadline is the right wording, most people refer to them as 'timeout'. So what about:

```suggestion
It's always reasonable to set a deadline (timeout) for a gRPC client, i.e. to specify a duration of time after which the RPC times out and the client receives the status error `DEADLINE_EXCEEDED`.
```

Reply:
Ok, np. I thought that the _"time after which the RPC times out.."_ in the following sentence is enough but sure, why not..